walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -32,6 +32,17 @@ private static boolean isAttended(Event event, MeetingRequest request) {
     return false; // Fallback for if no attendees are found that belong to both the event and the meeting request
   }
 
+  /**
+    * Returns the minutes in a day from a half hour slot
+    *
+    @param int halfHour The halfhour slot to be converted
+    @exception IllegalArgumentException thrown if slot > 0 or slot > 47.
+  */
+  private static int getTime(int halfHour) {
```

Comment:
getTime is a misleading naming.  Suggest to choose a name that describes what the method does, which would be something like "convertToMinuteTimestamp". You might find a better name along those lines.

Reply:
Method name updated with commi t437e712